src/redux/session.js

```diff
@@ -103,3 +214,52 @@ module.exports.refreshSession = () => (dispatch => {
         return;
     });
 });
+
+module.exports.handleLogIn = (formData, callback) => (dispatch => {
+    dispatch(module.exports.setLoginError(null));
+    formData.useMessages = true; // NOTE: this may or may not be being used anywhere else
```

Comment:
I think this comment should be removed or changed to explain what this is here for — prevent displaying the "Logged in" flash message on the next scratchr2 page that loads.